lib/util/api.ts

```diff
@@ -57,3 +57,17 @@ export function combineQueryParams(
   }
   return qs.stringify(search, { arrayFormat: 'repeat' })
 }
+
+/**
+ * Drops unused params so they don't show up in URL.
+ * TODO: Remove those params from core-utils.
+ */
+export function removeUnusedQueryParams(params: any): void {
```

Comment:
make the type `Record<string, any>`

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/1226/commits/b3f93e4cd9d3ff48df94812947e2084d112e5f95